### PR TITLE
Add CRUD orepations

### PR DESCRIPTION
**Pull request by Jossindev**

### CODE REVIEW

File: src/main/java/org/example/dto/SongDTO.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import lombok.extern.jackson.Jacksonized;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+
+@Builder
+@Getter
+@Setter
+@Jacksonized
```
Comment (havryliuk) [inline src/main/java/org/example/dto/SongDTO.java]: never seen such annotation, what does it do?
i thought spring boot supports json serialization-deserialization out of the box, at least i didn't have to do anything extra

File: src/main/java/org/example/model/Song.java [hunk 2]
```diff
@@ -4,16 +4,33 @@
...
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
 
 @Entity
-@Value
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```
Comment (havryliuk) [inline src/main/java/org/example/model/Song.java:17]: i think `@Data` annotation contains getter, setter and the constructors

**Comment by havryliuk** [comment]
looks good to me

File: src/main/java/org/example/dto/SongDTO.java (hunk 1 above)
Comment (bboiko-CLGX) [reply src/main/java/org/example/dto/SongDTO.java]: I use it sometimes when have problem with deserialization large object, or i need the exact name of filed in response.
Looks like in this case i do not need to use it.

**Pull request merged**